### ISSUE TITLE
Add overview of key features of Modelica to front page

### ISSUE THREAD

**Issue by t-sommer** [modelica/www.modelica.org#88]
with short examples, similar to https://www.python.org/.

![image](https://github.com/modelica/www.modelica.org/assets/2453858/afc98919-a952-45f2-9c97-70cfdda02e36)

### Equation Based

Define the equations and let the compiler do the work.

### Unit Safety

Connect components with confidence.

### Object Oriented

Create models with different levels of complexity.

### Modelica Standard Library

Comprehensive standard library with building blocks and examples from all physical domains.

### Graphical Modeling

Create and parameterize models in a graphical environment.

### Multi Domain

Simulation of multiple physical domains in one environment, e.g. thermal, electrical, mechanical, and logical.

**Comment by dzimmer** [comment]
I agree that this a good example.  The exact details will be always subject to discussion.  I will generate content so that we can realize this.
